class/Defaults/MergeTag/Post/PostThumbnailUrl.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Post thumbnail url merge tag
+ *
+ * Requirements:
+ * - Trigger property of the post type slug with WP_Post object
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\MergeTag\Post;
+
+use BracketSpace\Notification\Defaults\MergeTag\UrlTag;
+
+
+/**
+ * Post thumbnail url merge tag class
+ */
+class PostThumbnailUrl extends UrlTag {
+
+	/**
+	 * Post Type slug
+	 *
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * Merge tag constructor
+	 *
+	 * @since 5.0.0
+	 * @param array $params merge tag configuration params.
+	 */
+	public function __construct( $params = [] ) {
+
+		if ( isset( $params['post_type'] ) ) {
+			$this->post_type = $params['post_type'];
+		} else {
+			$this->post_type = 'post';
+		}
+
+		$args = wp_parse_args(
+			$params,
+			[
+				'slug'        => $this->post_type . '_thumbnail_url',
+				// translators: singular post name.
+				'name'        => sprintf( __( '%s thumbnail url', 'notification' ), $this->get_nicename() ),
+				'description' => __( 'https://example.com/wp-content/2019/01/image.jpg', 'notification' ),
+				'example'     => true,
+				'resolver'    => function() {
+					return wp_get_attachment_image_url( get_post_thumbnail_id( $this->trigger->{ $this->post_type }->ID ) );
```

Comment:
This will return a thumbnail, 300x300px default. This case can you add another merge tag called `FeaturedImageUrl` which will return full size?